index.js

```diff
@@ -4,11 +4,31 @@ const fetch = require('node-fetch');
 
 const key = '17c5c4c6e845e73f4296fa83b502b87c';
 
-const keyword = process.argv.splice(2).join(' ') || 'Big Yellow Door';
+const args = (process.argv.splice(2).join(' ') || 'Big Yellow Door@Delhi').split('@');
+
+if (args.length == 1) {
+  args.splice(1, 0, 'Delhi');
+} else if (args.length == 2) {
+  if (args[0] === '' || args[1] === '') {
+    console.log('Usage: zom <keyword>[@<location>]');
+    return;
+  }
+} else {
+  console.log('Usage: zom <keyword>[@<location>]');
+  return;
+}
+
+const { location, keyword } = { location: args[1], keyword: args[0] };
+
+const extractLocationData = e => new Promise((resolve, reject) => {
+  let locationData = e.location_suggestions[0] || { entity_id: 1, entity_type: 'city' }
+
+  resolve(locationData);
+});
```

Comment:
Let's use async await here :smile: 

```javascript

const extractLocationData = async e => (
  e.location_suggestions[0] || { entity_id: 1, entity_type: 'city' }
);

```

Reply:
Using `async` might limit the users able to use this cli. Let's keep it a bit compatible?

Reply:
We can always transpile. 

As per recent [survey](https://cdn.infoq.com/statics_s2_20170927-0419/resource/news/2017/07/node-2017-user-survey-results/en/resources/Screen%20Shot%202017-07-28%20at%2012.32.35%20AM.png) 57% are on v6, who might soon upgrade to version 8 when it hits LTS.

Thing is, since we are using a lot of promises already, it'll help us to use async-await. Either way, I'm open to both :smile: 

You can also use `Promise.resolve(value)` if you're planning to resolve right away.

Reply:
Let's not forget about the remaining ~43% of the folks. `Promise.resolve(value)` would be my pick, atleast for the time being ...